cmd/ui/src/views/GroupManagement/GroupManagement.tsx

```diff
@@ -61,10 +70,17 @@ const GroupManagement = () => {
                 label: openNode.name,
                 ...openNode,
             };
-            dispatch(searchbarActions.sourceNodeSelected(searchNode));
-            dispatch(setSelectedNode(openNode));
 
-            navigate(ROUTE_EXPLORE);
+            if (backButtonFlagQuery.data?.enabled) {
+                navigate({
+                    pathname: ROUTE_EXPLORE,
+                    search: `?selectedItem=${getGraphNodeByObjectId.data?.id}&searchType=node&primarySearch=${openNode?.id}`,
```

Comment:
This makes me think we could use some type safety when building these search queries! No need to change anything here, but im going to think about how we could build this search string a little easier

Reply:
Can we add `exploreSearchTab=node` here as well please?